README.md

```diff
@@ -120,7 +120,7 @@ Thanks goes to these wonderful people ([emoji key](https://github.com/kentcdodds
     <td align="center"><a href="https://github.com/nithin-pankaj"><img src="https://avatars2.githubusercontent.com/u/15152472?v=4" width="100px;" alt=""/><br /><sub><b>Nithin Pankaj</b></sub></a><br /><a href="https://github.com/fennecdjay/Gwion/commits?author=nithin-pankaj" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/evayde"><img src="https://avatars1.githubusercontent.com/u/25255815?v=4" width="100px;" alt=""/><br /><sub><b>Enrico Gruner</b></sub></a><br /><a href="https://github.com/fennecdjay/Gwion/commits?author=evayde" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/umer2001"><img src="https://avatars2.githubusercontent.com/u/35496058?v=4" width="100px;" alt=""/><br /><sub><b>Muhammad Umer Farooq</b></sub></a><br /><a href="https://github.com/fennecdjay/Gwion/commits?author=umer2001" title="Code">💻</a></td>
-    <td align="center"><a href="http://shuvalov.info"><img src="https://avatars2.githubusercontent.com/u/1410106?v=4" width="100px;" alt=""/><br /><sub><b>Shuvalov Anton</b></sub></a><br /><a href="https://github.com/fennecdjay/Gwion/commits?author=a" title="Documentation">📖</a></td>
+    <td align="center"><a href="http://shuvalov.info"><img src="https://avatars2.githubusercontent.com/u/1410106?v=4" width="100px;" alt=""/><br /><sub><b>Shuvalov Anton</b></sub></a><br /><a href="https://github.com/fennecdjay/Gwion/commits?author=a-mere-peasant" title="Documentation">📖</a></td>
```

Comment:
`href` should be https://github.com/a-mere-peasant.
`img src` should be https://avatars3.githubusercontent.com/u/50142453?s=400&u=4844f21d4f48431d854307ecdf4d1f4d80977ab9&v=4
The bold text should be ... well I don't know, let's ask @a-mere-peasant 
I hope the notation is clear.